tiledb/sm/filesystem/posix.h

```diff
@@ -35,23 +35,21 @@
 
 #ifndef _WIN32
 
-#include <ftw.h>
-#include <sys/types.h>
-
 #include <functional>
 #include <string>
 #include <vector>
 
-#include "tiledb/sm/buffer/buffer.h"
+#include <ftw.h>
```

Comment:
place this with the other system headers, in between line 38 and 39.

Reply:
I'll revert `<ftw.h>` and `<sys/types.h>` header files same as before in same location. Is it OK for you?

Reply:
I'm just saying that all headers included with the`<` and `>` characters 1) shouldn't be separated by new-lines and 2) sorted alphabetically.